.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           command: |
             PYTHONPATH=$PWD python -m coverage run -m pytest */testcases/integration_testcases/*.py */testcases/unit_testcases/*.py */testcases/unit_testcases/*/*.py */testcases/functional_testcases/*.py
             python -m coverage report -m
-            COVERALLS_REPO_TOKEN=${COVERALLS_REPO_TOKEN} python -m coveralls
+#            COVERALLS_REPO_TOKEN=${COVERALLS_REPO_TOKEN} python -m coveralls
```

Comment:
Why is this commented out?

Reply:
checked with Vinay as this was causing build issues in circleCLI , and was suggested to comment this out. @vinthedark please confirm

Reply:
The Coverall is throwing internal error whenever we are trying to publish the test results. So asked @DhivakharVenkatachalam to comment it for now to unblock the merge.

Reply:
failed build link because of coverall internal error https://app.circleci.com/pipelines/github/singnet/snet-marketplace-service/4156/workflows/26f1c730-9ddd-4a72-97c7-12b5db47cd5c/jobs/4224